<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SiBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 219, 173);
border-color: rgb(176, 121, 70);
font: 8pt &quot;Snap ITC&quot;;</string>
  </property>
  <widget class="QTabWidget" name="tab_az">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>791</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>АБ</string>
    </attribute>
    <widget class="QTableWidget" name="table_ab">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ВГ</string>
    </attribute>
    <widget class="QTableWidget" name="table_vg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>ДЕ</string>
    </attribute>
    <widget class="QTableWidget" name="table_de">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>ЖЗИЙ</string>
    </attribute>
    <widget class="QTableWidget" name="table_jy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>КЛ</string>
    </attribute>
    <widget class="QTableWidget" name="table_kl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>МН</string>
    </attribute>
    <widget class="QTableWidget" name="table_mn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>ОП</string>
    </attribute>
    <widget class="QTableWidget" name="table_op">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>РС</string>
    </attribute>
    <widget class="QTableWidget" name="table_rs">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>ТУ</string>
    </attribute>
    <widget class="QTableWidget" name="table_tu">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>ФХ</string>
    </attribute>
    <widget class="QTableWidget" name="table_fh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_11">
    <attribute name="title">
     <string>ЦЧШЩ</string>
    </attribute>
    <widget class="QTableWidget" name="table_csh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>ЪЫЬЭ</string>
    </attribute>
    <widget class="QTableWidget" name="table_uh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_13">
    <attribute name="title">
     <string>ЮЯ</string>
    </attribute>
    <widget class="QTableWidget" name="table_xyz">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_14">
    <attribute name="title">
     <string>A-Z</string>
    </attribute>
    <widget class="QTableWidget" name="table_az">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>731</width>
       <height>601</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(21, 22, 24); 
font: 8pt &quot;Sitka Heading&quot;; font-size:16pt;</string>
   </property>
   <property name="text">
    <string>Зашли как:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>233</width>
     <height>215</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="refresh_but">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 12pt;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_contact_but">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 12pt;</string>
      </property>
      <property name="text">
       <string>Add contact</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="change_contact_but">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 12pt;</string>
      </property>
      <property name="text">
       <string>Changed contact</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_contact_but">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 12pt;</string>
      </property>
      <property name="text">
       <string>Delete contact</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_but">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 12pt;</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="account_label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
